astropy/coordinates/tests/test_sky_coord_velocities.py

```diff
@@ -17,7 +17,7 @@
     SphericalRepresentation, UnitSphericalCosLatDifferential, UnitSphericalDifferential,
     UnitSphericalRepresentation)
 from astropy.tests.helper import assert_quantity_allclose
-from astropy.utils.compat.optional_deps import HAS_SCIPY  # noqa
+from astropy.utils.compat.optional_deps import HAS_SCIPY  # noqa: F401
```

Comment:
Maybe cases like this mean that we should write ``skipif`` decorators using actual Python code instead of strings?

Reply:
I think this would be better if it would work.